SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -862,6 +864,13 @@
         "ceph"
       ]
     },
+    {
+      "origin": "netplan",
+      "license": "[GPLv3](https://github.com/canonical/netplan/blob/main/COPYING)",
```

Comment:
just to be sure...the netplan SPEC file originated from netplan and not fedora, or centos or something?

Reply:
It is from Netplan sources. 
Here is the reference:  https://github.com/canonical/netplan/blob/main/rpm/netplan.spec